tools/cxbe/Xbe.cpp

```diff
@@ -433,8 +435,17 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, bool x_bRetail)
             }
         }
 
-        // TODO: make room for library versions
+        // make room for library versions
         {
+            if (m_Header.dwLibraryVersions == 0)
+            {
+                m_Header.dwLibraryVersionsAddr = 0;
+            }
+            else
+            {
+                m_Header.dwLibraryVersionsAddr = mrc;
+                mrc += m_Header.dwLibraryVersions * static_cast<uint32>(sizeof(*m_LibraryVersion));
```

Comment:
nit: Codebase seems to use C style casts.

We should eventually clean the entire code up.
The whole allocation logic and code style is horrible anyway.